hide only rule dialogue turns (#7701)
RasaHQ/rasa: changelog/7701.improvement.md

> Hide dialogue turns predicted by `RulePolicy` in the tracker states
> for ML only policies like `TEDPolicy`
> if dialogue turns only appear as a rule in the training data and does not appear in stories.

Reviewer: ```suggestion
Hide dialogue turns predicted by `RulePolicy` in the tracker states
for ML-only policies like `TEDPolicy`
if those dialogue turns only appear as rules in the training data and do not appear in stories.
```
